gcloud/_apitools/test_util.py

```diff
@@ -0,0 +1,77 @@
+# pylint: skip-file
+import unittest2
+
+
+class Test_TypeCheck(unittest2.TestCase):
```

Comment:
Frankly, I wouldn't bother:  the tests are all straightforward "reverse engineering", just so that we can know what breaks when we start refactoring.